tests/rpz/test.sh

```
@@ -0,0 +1,74 @@
+    rlPhaseStartTest "RPZ with master only"
+        rlRun "podman cp ./dnsconfd.conf $dnsconfd_cid://etc/dnsconfd.conf" 0 "Copying RPZ config into container"
+        rlRun "podman exec $dnsconfd_cid systemctl restart dnsconfd" 0 "Restarting dnsconfd with RPZ config"
+        rlRun "podman exec $dnsconfd_cid nmcli connection mod eth0 ipv4.dns 192.168.6.10" 0 "Adding dns server to NM active profile"
+        rlRun "podman exec $dnsconfd_cid nmcli connection up eth0" 0 "Bringing the connection up"
+        # allow time for RPZ zone transfer from BIND to Unbound
+        sleep 5
+        rlRun "podman exec $dnsconfd_cid getent hosts address.test.com | grep 192.168.6.10" 0 "Verifying non-blocked address resolves"
+        rlRun "podman exec $dnsconfd_cid getent hosts blocked.test.com | grep 192.168.6.10" 1 "Verifying RPZ-blocked address does not resolve"
+        rlRun "podman exec $dnsconfd_cid dig blocked.test.com | grep NXDOMAIN" 0 "Verifying RPZ returns NXDOMAIN for blocked domain"
+        # dynamically add address.test.com to the RPZ block list on BIND
+        rlRun "podman exec $bind_cid /bin/bash -c 'printf \"server 127.0.0.1\nzone rpz.dnsconfd.test\nupdate add address.test.com.rpz.dnsconfd.test. 86400 CNAME .\nsend\n\" | nsupdate'" 0 "Adding address.test.com to RPZ zone via nsupdate"
```

Shellcheck (0.11.0)

[warning] bind_cid is referenced but not assigned.

(SC2154)